tools/hubl/internal/keyring.go

```diff
@@ -96,19 +97,28 @@ func KeyringCmd(chainName string) *cobra.Command {
 				return err
 			}
 
-			addressCodec, validatorAddressCodec, consensusAddressCodec, err := getAddressCodecFromConfig(cfg, chainName)
+			// addressCodec, validatorAddressCodec, consensusAddressCodec, err := getAddressCodecFromConfig(cfg, chainName)
```

Comment:
I keep the comment around to remember to switch this when we'll use v0.51.

Reply:
Can we add a reference issue or something?

Reply:
I've cross referenced it here: https://github.com/cosmos/cosmos-sdk/issues/18417